README.md

```diff
@@ -1,44 +1,68 @@
 # pwned-password-check-net
-Simple .net core command line utility for checking your pwned passwords against Troy Hunt's Have I Been Pwned? service.
+Simple .net core command line utility for checking your pwned accounts against Troy Hunt's Have I Been Pwned? service. Export accounts into CSV file and run pwned-password-check-net.dll.
 
+> dotnet pwned-password-check-net.dll -p accounts.csv -l KeePass
 
-# Usage
-## Build from source code
+# Install
+## Windows
+### Build from source code
 Clone this repository and compile with dotnet tooling. (or visual studio code)
-## Download
+### Download
 Download from AppVeyor continuous integration.
-[Latest build](https://ci.appveyor.com/api/projects/hopp/pwned-password-check-net/artifacts/pwned-password-check-net.zip?bramch=master)
-
-## Export passwords
-Export your passwords to csv file, with structure
-
-| Account | LoginName | Password | WebSite |
-| ---     | --- | ---| --- |
-| My favourite service|your_account@your_domain.com|password|www.myfavouriteservice.nowhere
-This is default export from KeePass.
-## Run
-There are two parameters
-|Paremeter|Long Name|Description|
+[Latest build](https://ci.appveyor.com/api/projects/hopp/pwned-password-check-net/artifacts/pwned-password-check-net.zip?branch=master)
+
+## Linux
+    sudo snap install dotnet-sdk --classic
```

Comment:
This is maybe matter of taste. But I do prefer to have .net core installed without snap.
https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1904